qa/workunits/rbd/cli_migration.sh

```diff
@@ -199,6 +218,47 @@ EOF
     remove_image "${dest_image}"
 }
 
+# Function to test the import of native format with namespace
+test_import_native_format_with_namespace() {
+    local base_namespace=$1
+    local base_image=$2
+    local dest_namespace=$3
+    local dest_image=$4
+
+    local pool_id=$(ceph osd pool ls detail --format xml | xmlstarlet sel -t -v "//pools/pool[pool_name='rbd']/pool_id")
+    cat > ${TEMPDIR}/spec.json <<EOF
+{
+  "type": "native",
+  "pool_id": ${pool_id},
+  "pool_namespace": "${base_namespace}",
+  "image_name": "${base_image}",
+  "snap_name": "2"
+}
+EOF
+    cat ${TEMPDIR}/spec.json
+
+    # Migration from namespace to non-namespace
+    rbd migration prepare --import-only \
+        --source-spec-path ${TEMPDIR}/spec.json ${dest_image}
+    rbd migration execute ${dest_image}
+    rbd migration commit ${dest_image}
+
+    compare_images "rbd/${base_namespace}/${base_image}@1" "${dest_image}@1"
+    compare_images "rbd/${base_namespace}/${base_image}@2" "${dest_image}@2"
+
+    # Migration from namespace to namespace
+    rbd namespace create rbd/${dest_namespace}
```

Comment:
I'd suggest creating this namespace at the top level as well, together with `rbd/${NAMESPACE1}`.

Reply:
Done